### PR TITLE
Add code style enforcement, code of conduct

### PR DESCRIPTION
**Pull request by jtenner**
@hustcc We should pull this before releasing 2.3.0.

Any thoughts?

### CODE REVIEW

**Comment by jtenner** [comment]
Also, shouldn't we setup some travis/github action ci build processes?

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 language: node_js
```
Comment (hustcc) [inline .travis.yml:1]: use github action?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,8 +2,12 @@
 
 ## Version 2.3.0
 
+- Added Prettier code style
+- Deleted .npmignore and switched to `package.json` files field
+- Added [CONTRIBUTING](./CONTRIBUTING.md) and [CODE_OF_CONDUCT](./CODE_OF_CONDUCT.md) docs
 - Switched to `moo-color` for color parsing
```
Comment (hustcc) [inline CHANGELOG.md:8]: why switch to a new project?

File: README.md [hunk 3]
```diff
@@ -175,10 +175,10 @@ canvas.toDataURL.mockReturnValueOnce(
 
 ## Contributors
 
-* [@hustcc](https://github.com/hustcc)
-* [@jtenner](https://github.com/jtenner)
-* [@evanoc0](https://github.com/evanoc0)
+- [@hustcc](https://github.com/hustcc)
```
Comment (hustcc) [inline README.md:178]: Contributors  all is here, https://github.com/hustcc/jest-canvas-mock/graphs/contributors

Maybe a table is more beautiful.

File: CHANGELOG.md (hunk 2 above)
Comment (jtenner) [reply CHANGELOG.md:8]: `moo-color` supports the `"transparent"` color.

File: README.md (hunk 3 above)
Comment (jtenner) [reply README.md:178]: I can redo the contributors too. I just wanted to add the list to the readme. Is that okay?

File: .travis.yml (hunk 1 above)
Comment (jtenner) [reply .travis.yml:1]: It's just something to think about. Do you think it would be a good idea?

File: CHANGELOG.md (hunk 2 above)
Comment (hustcc) [reply CHANGELOG.md:8]: `moo-color` is written by  contributor? I'm worried about compatibility.

Maybe the best way is to send pr to `parse-color` for support.

File: .travis.yml (hunk 1 above)
Comment (hustcc) [reply .travis.yml:1]: Year, GitHub action is excellent, but change to GitHub is not an emergency.

File: README.md (hunk 3 above)
Comment (hustcc) [reply README.md:178]: Of course that is okay!

File: README.md [hunk 4]
```diff
@@ -175,10 +175,10 @@ canvas.toDataURL.mockReturnValueOnce(
...
+- [@jtenner](https://github.com/jtenner)
```
Comment (hustcc) [inline README.md:179]: move yourself to first by the comment and contribute.

File: CHANGELOG.md (hunk 2 above)
Comment (jtenner) [reply CHANGELOG.md:8]: I checked the source of `moo-color`. It's very well done. I think it's okay for compatibility.  Doesn't travis already test node 8?

File: README.md (hunk 4 above)
Comment (jtenner) [reply README.md:179]: You are the primary contributor. This was *your* idea and *your* repo in the first place. I merely filled in the details and assertions.

I don't need to be placed primary on the list.

File: CHANGELOG.md (hunk 2 above)
Comment (jtenner) [reply CHANGELOG.md:8]: Also, `parse-color` hasn't been updated in 8 years, but `moo-color` still has active support. I think this is a non-issue.

Comment (hustcc) [reply CHANGELOG.md:8]: ok~

**Pull request merged**